### ISSUE TITLE
Compatibility with rtweet 1.0.0

### ISSUE THREAD

**Issue by maelle** [ropensci/tidytags#60]
In the dashboard of all rOpenSci packages I notice an error when building tidytags for rtweet https://ropensci.r-universe.dev/ui#builds

```r
> test_check("tidytags")
  Saving auth to 'C:\Users\runneradmin\AppData\Roaming/R/config/R/rtweet/create_token.rds'
  Error in eval(exprs, env) : object 'load_tokens' not found
  Calls: test_check ... source_test_helpers -> source_dir -> lapply -> FUN -> eval -> eval
  In addition: Warning message:
  `create_token()` was deprecated in rtweet 1.0.0.
  See vignette('auth') for details
```

I'm assuming it might be related to rtweet development version which is the one used in ropensci r-universe.

It might be good to think about before CRAN submission of tidytags (after which tidytags will be a CRAN reverse dependency of rtweet cc @llrs)

**Comment by llrs** [comment]
It is related to using an internal rtweet function that is no longer on rtweet devel: https://github.com/ropensci/tidytags/blob/6f25094f19a9a39fd8ead390f15354b7810a0392/tests/testthat/helper-vcr.R#L35
I think that so far the changes have been deprecations and a couple of defunct functions, but they should have their own error message.

**Comment by bretsw** [comment]
Thank you, @maelle, for alerting me to this. And thanks @llrs for further insight. I'm looking into this now.

**Comment by bretsw** [comment]
Ok, some immediate confusion... The error message said: "`create_token()` was deprecated in rtweet 1.0.0." But rtweet 1.0.0 hasn't been released anywhere yet. And the updated authentication information is for the retweet 0.7.0.9011 dev version. How much should we expect tidytags users to be getting into dev versions of other packages? 

Thanks @maelle and @llrs for any further insight here.

**Comment by maelle** [comment]
Right, it's tricky because on CRAN you'll get CRAN rtweet. So maybe fine to keep things as is now? :thinking:

**Comment by bretsw** [comment]
@llrs, any guess on when rtweet 1.0.0 is coming? 

I'm excited about the changes in auth settings... I just don't want to get ahead of myself.

**Comment by llrs** [comment]
I have a package that it is on a similar spot as tidytags, What I did was check the version of the package and depending on that use some functions or others. However, you shouldn't use any internal function of any package, as those might change even between minor versions (on CRAN and elsewhere). 

I have made several guesses before that wen completely off, my current estimate is be ready by summer. But that also depends on how prepared will be dependencies to the new version of rtweet and how goes the CRAN submission process.
